### PR TITLE
update cli readme per change in clicommon

### PR DESCRIPTION
**Pull request by RodgeFu**
### Latest improvements:
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i><br>
### Contribution checklist:
- [ ] I have reviewed the [documentation](https://github.com/Azure/adx-documentation-pr/wiki/Overall-basic-flow) for the workflow.
- [ ] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
- [ ] The [OpenAPI Hub](https://aka.ms/openapiportal) was used for checking validation status and next steps.
### ARM API Review Checklist
- [ ] Service team MUST add the "WaitForARMFeedback" label if the management plane API changes fall into one of the below categories. 
- adding/removing APIs.
- adding/removing properties.
- adding/removing API-version. 
- adding a new service in Azure.

Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.
- [ ] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://armwiki.azurewebsites.net/rp_onboarding/ResourceProviderOnboardingAPIRevieworkflow.html).

### CODE REVIEW

**Comment by RodgeFu** [comment]
purely codegen change in xxx.cli.md. Assign to Qiaoqiao to review. thanks.

File: specification/applicationinsights/resource-manager/readme.cli.md [hunk 1]
```diff
@@ -15,17 +15,17 @@ cli:
       - select: 'operation'
         where:
             operationGroup: 'components'
-            operation: '^(delete|purge|get_purge_status)$'
+            operation: '^(delete|purge|getPurgeStatus)$'
```
Comment (qiaozha) [inline specification/applicationinsights/resource-manager/readme.cli.md]: Should operation start with Capital Letter?

File: specification/applicationinsights/resource-manager/readme.cli.md [hunk 2]
```diff
@@ -15,17 +15,17 @@ cli:
...
         removed: true
       - select: 'operation'
         where:
-            operationGroup: 'API_keys'
+            operationGroup: 'APIkeys'
             operation: 'create|delete'
```
Comment (qiaozha) [inline specification/applicationinsights/resource-manager/readme.cli.md]: like wise?

File: specification/applicationinsights/resource-manager/readme.cli.md [hunk 3]
```diff
@@ -15,17 +15,17 @@ cli:
...
         removed: true
       - select: 'operation'
         where:
             operationGorup: 'components'
-            operation: 'update_tags'
+            operation: 'updateTags'
```
Comment (qiaozha) [inline specification/applicationinsights/resource-manager/readme.cli.md]: likewise?

File: specification/blockchain/resource-manager/readme.cli.md [hunk 4]
```diff
@@ -23,24 +23,24 @@ cli:
...
       - select: 'operation'
         where:
-            operationGroup: 'blockchain_members'
+            operationGroup: 'blockchainMembers'
             operation: '(create|update)'
         hidden: true
       - select: 'operation'
         where:
-            operationGroup: 'transaction_nodes'
+            operationGroup: 'transactionNodes'
```
Comment (qiaozha) [inline specification/blockchain/resource-manager/readme.cli.md]: Should the operationGroup also startWith a Capital Letter

**Review by qiaozha — APPROVED** [review]

**Review by qiaozha — CHANGES_REQUESTED** [review]
operation and operationGroup in directive should start with a capital latter ?

**Comment by RodgeFu** [comment]
> operation and operationGroup in directive should start with a capital latter ?

actually it doesnt matter, it's case insensitive now

File: specification/applicationinsights/resource-manager/readme.cli.md (hunk 1 above)
Comment (RodgeFu) [reply specification/applicationinsights/resource-manager/readme.cli.md]: it's case insensitive now

File: specification/applicationinsights/resource-manager/readme.cli.md (hunk 2 above)
Comment (RodgeFu) [reply specification/applicationinsights/resource-manager/readme.cli.md]: it's case insensitive now

File: specification/applicationinsights/resource-manager/readme.cli.md (hunk 3 above)
Comment (RodgeFu) [reply specification/applicationinsights/resource-manager/readme.cli.md]: it's case insensitive now

**Comment by qiaozha** [comment]
> > operation and operationGroup in directive should start with a capital latter ?
> 
> actually it doesnt matter, it's case insensitive now



> > operation and operationGroup in directive should start with a capital latter ?
> 
> actually it doesnt matter, it's case insensitive now

I see. Can you take a look at the go sdk failure? @NullMDR

**Review by qiaozha — APPROVED** [review]

**Pull request merged**